src/SIL.LCModel/Templates/NewLangProj.fwdata

```diff
@@ -5929,6 +5929,7 @@
 <AUni ws="en">ConChrtTempl</AUni>
 </Abbreviation>
 <Depth val="127" />
+<IsSorted val="true" />
```

Comment:
This line changes the way templates are sorted from user-determined to alphabetical.